.eslintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "env": {
+    "node": true,
+    "es6": true,
+    "mocha": true
```

Comment:
jest ?

Reply:
The boilerplate use mocha. For the moment jest is not in the project. But Yes, we can use it later.